### DISCUSSION TITLE
How do I know which ES features a specific version of `@babel/core` supports?

### DISCUSSION THREAD

**Discussion by leonardoraele** [babel/babel#14142] [Q&A] [answered]
Is there a feature matrix somewhere out there? Something similar to https://node.green/?

**Comment by nicolo-ribaudo** [comment, marked as answer]
By itself `@babel/core` doesn't support any specific ECMAScript feature, but you can expand the "History" section at https://babeljs.io/docs/en/babel-parser#ecmascript-proposalshttpsgithubcombabelproposals, to see the features supported by the different `@babel/parser` versions. Note that `@babel/core` depends on `@babel/parser` using `^` in the version, so if you install `@babel/core@7.0.0` now you will still get the last `@babel/parser` version.

Another very good resource is https://babeljs.io/docs/en/features-timeline (I just noticed we need to update it to 7.16).

As an alternative, you can check our full changelog at https://github.com/babel/babel/blob/main/CHANGELOG.md
